ideascube/widgets.py

```diff
@@ -1,6 +1,56 @@
 from django.forms import widgets
 
 from ideascube.configuration import get_config
+from django.utils.html import mark_safe, escape
+from django.utils.encoding import force_text
+from django.forms.utils import flatatt
+
+
+_script_template = '''<script type="text/javascript">tinymce.init({{
+selector: "#{widget_id}",
+inline: true,
+menubar: {menubar},
+toolbar: {toolbar},
+plugins: "{plugins}",
+hidden_input: false
+}});</script>'''
+
+
+class RichTextEntry(widgets.Widget):
+    def __init__(
+        self, menubar=False, toolbar=False, plugins="autolink", attrs=None):
+        super().__init__(attrs=attrs)
+        self.menubar = menubar
+        self.toolbar = toolbar
+        self.plugins = plugins
+
+    def render(self, name, value, attrs=None):
+        if value is None:
+            value = ''
+
+        # The widget displayed to the user
+        div_attrs = {'class': 'tinymce_editable',
+                        'id': name}
+        div_html = '<div{}>{}</div>'.format(
+            flatatt(div_attrs),
+            force_text(value)
+        )
+
+        # The input used by django
+        input_attrs = self.build_attrs(attrs, name=name, value=escape(value), type='hidden')
+        input_html = '<input{} />'.format(
+            flatatt(input_attrs)
+        )
+
+        # Make tinymce handle the widget
+        script_html = _script_template.format(
+            widget_id = name,
+            menubar = 'true' if self.menubar else 'false',
+            toolbar = 'true' if self.toolbar else 'false',
+            plugins = self.plugins)
```

Comment:
The main problem I've seen is for the future.
If we want to properly handle images/media in the blog application (select them from what we have in the mediacenter), we will have to pass custom js function to tinymce.
django-tinymce do so nasty string `eval` to handle this point. Even if I agree that those `<script>` in the middle of the page is not perfect, it still seems better that handle complex this in js latter.